[2025-10-09T10:57:29.316+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-09T10:57:29.371+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_flow.check_users_freshness scheduled__2025-09-28T00:00:00+00:00 [queued]>
[2025-10-09T10:57:29.384+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_flow.check_users_freshness scheduled__2025-09-28T00:00:00+00:00 [queued]>
[2025-10-09T10:57:29.386+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-09T10:57:29.410+0000] {taskinstance.py:2330} INFO - Executing <Task(ShortCircuitOperator): check_users_freshness> on 2025-09-28 00:00:00+00:00
[2025-10-09T10:57:29.422+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=214) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-09T10:57:29.424+0000] {standard_task_runner.py:64} INFO - Started process 220 to run task
[2025-10-09T10:57:29.421+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline_flow', 'check_users_freshness', 'scheduled__2025-09-28T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpogu7r6j_']
[2025-10-09T10:57:29.426+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask check_users_freshness
[2025-10-09T10:57:29.532+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline_flow.check_users_freshness scheduled__2025-09-28T00:00:00+00:00 [running]> on host 321597a50810
[2025-10-09T10:57:29.758+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='oluwaseyiogundimu10@gmail.com' AIRFLOW_CTX_DAG_OWNER='seyi' AIRFLOW_CTX_DAG_ID='pipeline_flow' AIRFLOW_CTX_TASK_ID='check_users_freshness' AIRFLOW_CTX_EXECUTION_DATE='2025-09-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-28T00:00:00+00:00'
[2025-10-09T10:57:29.761+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-09T10:57:29.795+0000] {baseoperator.py:400} WARNING - ShortCircuitOperator.execute cannot be called outside TaskInstance!
[2025-10-09T10:57:29.814+0000] {s3_client.py:23} INFO - Successfully initialized S3 client.
[2025-10-09T10:57:30.672+0000] {logging_mixin.py:188} INFO - users.csv is fresh: 2025-10-09 10:57:26+00:00
[2025-10-09T10:57:30.677+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-09T10:57:30.681+0000] {python.py:304} INFO - Condition result is None
[2025-10-09T10:57:30.683+0000] {python.py:331} INFO - Skipping downstream tasks
[2025-10-09T10:57:30.745+0000] {python.py:339} INFO - Done.
[2025-10-09T10:57:30.746+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-09T10:57:30.764+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_flow, task_id=check_users_freshness, run_id=scheduled__2025-09-28T00:00:00+00:00, execution_date=20250928T000000, start_date=20251009T105729, end_date=20251009T105730
[2025-10-09T10:57:30.809+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-09T10:57:30.838+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-09T10:57:30.843+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
